cmd/kubeadm/app/util/audit/utils.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package audit
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	auditv1beta1 "k8s.io/apiserver/pkg/apis/audit/v1beta1"
+
+	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
+	"k8s.io/kubernetes/cmd/kubeadm/app/util"
+
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+// CreateDefaultAuditLogPolicy writes the default audit log policy to disk.
+func CreateDefaultAuditLogPolicy(policyDir string) error {
+	policy := auditv1beta1.Policy{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "audit.k8s.io/v1beta1",
+			Kind:       "Policy",
+		},
+		Rules: []auditv1beta1.PolicyRule{
+			{
+				Level: auditv1beta1.LevelMetadata,
```

Comment:
Users cannot customize the rules here.

Reply:
How do you propose we expose?

Reply:
@chuckha @timothysc Let the users use their own policy files optionally.
Just make a slight change to current `AuditPolicyDir` behavior.
If `AuditPolicyDir` is not set, create and use this default policy file. If set, find that file named `audit.yaml` and use it.

Reply:
sgtm!